soap/soap.go

```diff
@@ -144,32 +144,43 @@ type basicAuth struct {
 }
 
 type options struct {
-	tlsCfg      *tls.Config
-	auth        *basicAuth
-	timeout     time.Duration
-	contimeout  time.Duration
+	tlsCfg           *tls.Config
+	auth             *basicAuth
+	timeout          time.Duration
+	contimeout       time.Duration
```

Comment:
Let's follow Go's naming conventions: 

```suggestion
	connTimeout       time.Duration
```

Reply:
I think that is out of scope of this pull request. In general, the code doesn't follow go format or linting guidelines and I suggest we did both in two separate pull requests.

Reply:
fair enough.

Reply:
Fair enough.

Reply:
Thanks for the suggestions. I ran `go lint` on the code locally but it seems we would need to change public contracts due to violations such as `soap/soap.go:126:2: struct field XmlNSWsse should be XMLNSWsse`. It might be worth investigating a tool like gometalinter/golangci-lint and having exclusions so we can add support.

As mentioned, for me the first step is to get `go fmt` checking which I've added here https://github.com/hooklift/gowsdl/pull/127 your feedback is appreciated.